pkg/apis/monitoring/v1/types.go

```diff
@@ -579,7 +579,7 @@ type RemoteWriteSpec struct {
 	WriteRelabelConfigs []RelabelConfig `json:"writeRelabelConfigs,omitempty"`
 	//BasicAuth for the URL.
 	BasicAuth *BasicAuth `json:"basicAuth,omitempty"`
-	// File to read bearer token for remote write.
+	// bearer token for remote write.
```

Comment:
Nit but let's capitalize so we are consistent with the rest:
```suggestion
	// Bearer token for remote write.
```